api/core/model_runtime/model_providers/tencent/speech2text/flash_recognizer.py

```diff
@@ -0,0 +1,158 @@
+import base64
+import hashlib
+import hmac
+import time
+
+import requests
+
+
+class Credential:
+    def __init__(self, secret_id, secret_key):
+        self.secret_id = secret_id
+        self.secret_key = secret_key
+
+
+# 录音识别极速版使用

```

Comment:
To facilitate community co-creation, please change the comment to English, thank you!

Reply:
flash_recognizer is copied from tencent SDK

Reply:
fixed